esmvalcore/experimental/templates/head.j2

```diff
@@ -70,4 +73,46 @@
       margin-right: 1em;
     }
   </style>
+{# 
+    .div_figure {
+      display: inline-block;
+    }
+
+      /* Style the tab */
+    .tab {
+      overflow: hidden;
+      border: 1px solid #ccc;
+      background-color: #f1f1f1;
+    }
+
+    /* Style the buttons that are used to open the tab content */
+    .tab button {
```

Comment:
The advantage of using a framework like bootstrap over writing your own stylesheets is that it is 1) less work and 2) looks better (unless you put in a lot of effort of course ;-). Would it work to replace this here by e.g. https://www.w3schools.com/bootstrap/bootstrap_buttons.asp?